package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "passport-saml-metadata",
-  "version": "2.4.1",
+  "version": "2.5.0",
```

Comment:
Please revert this change, I'll handle bumping versions on release.

Reply:
Yes sorry about that, I can test authentication on beginn of next week I guess.
 I can confirm all tests are passing right now:

> Executing task: npm run test <


> passport-saml-metadata@2.4.1 test
> npm run lint && npm run coverage


> passport-saml-metadata@2.4.1 lint
> semistandard src/*.js src/**/*.js test/*.js test/**/*.js --fix


> passport-saml-metadata@2.4.1 coverage
> nyc npm run mocha


> passport-saml-metadata@2.4.1 mocha
> mocha --recursive --timeout 15000 --exit



  fetch()
    ✔ loads
    ✔ fetches metadata XML from URL
    ✔ fetches metadata XML from backupStore
    ✔ fails with short timeout and empty backupStore
    ✔ accepts custom axios instances

  passport helpers
    ✔ toPassportConfig()
    ✔ toPassportConfig() with multiple cert support
    ✔ claimsToCamelCase()

  MetadataReader
    ✔ loads
    exposes property
      ✔ identityProviderUrl
      ✔ logoutUrl
      ✔ identifierFormat
      ✔ encryptionCert
      ✔ signingCert
      ✔ claimSchema
    handles no KeyDescriptor use attribute
      ✔ encryptionCert
      ✔ signingCert
    parses correctly multiline certificate key
      ✔ encryptionCert
      ✔ signingCert
    supports alternate authnRequestBinding HTTP-POST
      ✔ identityProviderUrl
      ✔ logoutUrl
    supports alternate authnRequestBinding HTTP-Artifact
      ✔ identityProviderUrl
      ✔ logoutUrl


  23 passing (92ms)

-------------|---------|----------|---------|---------|-----------------------------
File         | % Stmts | % Branch | % Funcs | % Lines | Uncovered Line #s           
-------------|---------|----------|---------|---------|-----------------------------
All files    |   73.23 |    31.58 |     100 |   73.23 |                             
 fetch.js    |   90.63 |    57.14 |     100 |   90.63 | 35-36,38                    
 passport.js |   91.67 |       75 |     100 |   91.67 | 26                          
 reader.js   |   63.86 |    18.52 |     100 |   63.86 | ...,154-157,172-173,179-182 
-------------|---------|----------|---------|---------|-----------------------------